1.8/tutorial/resources.md

```diff
@@ -0,0 +1,83 @@
+---
+post_title: DC/OS 101 - Understanding Ressources
+menu_order: 3
+---
+
+# Prerequisites
+We expect you to have access to a running DC/OS OSS cluster and the DC/OS CLI installed and configured.
+Furthermore, you have just deployed app2 in our cluster and have verified it is running.
+
+# Objective
+When installing all these apps, we can see that the overall resource utilization in the UI goes up.
+But what does increased resource utilization mean happen, how are resource limits enforced. How can I debug issues arising in the context of resource utilization?
+
+# Steps
+Resource Mangement and resource isolation between task is one of the core challenges of an operating system to manage resources.
```

Comment:
+Resource Mangement and resource isolation between tasks are core challenges for an operating system, which needs to manager resources.